politeiawww/pi/pi.go

```diff
@@ -40,6 +40,30 @@ func (p *Pi) HandlePolicy(w http.ResponseWriter, r *http.Request) {
 	util.RespondWithJSON(w, http.StatusOK, p.policy)
 }
 
+// HandleSummaries is the request handler for the pi v1 Summaries route.
+func (p *Pi) HandleSummaries(w http.ResponseWriter, r *http.Request) {
```

Comment:
Same comment as before.  Order the functions in the same order that they appear in the API.